doc/source/getting_started.rst

```diff
@@ -0,0 +1,190 @@
+   ``native_regions`` and ``common_regions`` are fixed.
+-  Required properties are ``model`` and **at least** either
+   ``native_regions`` or ``common_regions``. **Both** are **allowed** as
+   well.
+-  **``model``** (str): specifies the model name for which the mapping
+   applies.
+-  **``native_regions``** (list): list of model native regions serves as
+   a selection as to which regions to keep.
+
+   -  In the above example ``region_a`` is to be renamed to
+      ``alternative_name_a``. This is done by defining a key-value pair
+      of ``model_native_name: new_name``.
+   -  ``region_b`` is selected but the name is not changed.
+   -  Assuming ``model_a`` also defines a third region ``region_c``,
+      since it is not mentioned it will be **dropped** from the data.
+
+-  **``common_regions``** (list): list of common regions which will be
+   computed as aggregates. They are defined as list entries which
+   themselves have a list of constituent regions. These constituent
+   regions must be model native regions.
+
+   -  **Important to note** the names of the constituent regions
+      **must** refer to the **original** model native region names. In
+      the above example ``region_a`` and ``region_b`` and **not**
+      ``alternative_name_a``.
+
+Guidelines and naming convention
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The variable name (code) should adhere to the following conventions:
+
+-  A ``|`` (pipe) character indicates levels of hierarchy
+-  Do not use spaces before and after the ``|`` character, but add a
+   space between words (e.g., ``Primary Energy|Non-Biomass Renewables``)
+-  All words must be capitalised (except for ‘and’, ‘w/’, ‘w/o’, etc.)
+-  Do not use abbreviations (e.g, ‘PHEV’) unless strictly necessary
+-  Add hierarchy levels where it might be useful in the future, e.g.,
+   use ``Electric Vehicle|Plugin-Hybrid`` instead of ‘Plugin-Hybrid
+   Electric Vehicle’
+-  Do not use abbreviations of statistical operations (‘min’, ‘max’,
+   ‘avg’) but always spell out the word
+-  Do not include words like ‘Level’ or ‘Quantity’ in the variable,
+   because this should be clear from the context or unit
+
+Regions
+~~~~~~~
+
+The *region* codelist of the nomenclature will be read from all yaml
+files located in a folder of that name (including any sub-folders). To
+avoid repeating a “hierarchy” attribute many times (e.g., country,
+continent), the yaml files must have a nested dictionary structure:
+
+.. code:: yaml
+
+   - <Hierarchy Level>:
+     - Region Name:
+         Attribute: Attribute value
+
+When importing the codelist, the hierarchy will be added as attribute,
+such that it can be retrieved as
+
+.. code:: python
+
+   DataStructureDefinition.region["Region Name"]["Hierarchy"] = "<Hierarchy Level>"
+
+Other attributes specified in the yaml file can include (for countries)
+ISO2/3-codes, or the list of countries included in a macro-region (i.e.,
+a continent or large region).
+
+The pyam package
```

Comment:
Don't need to include the advertisement section for pyam here...

Reply:
Done.